x-pack/plugins/observability_ai_assistant/server/routes/chat/route.ts

```diff
@@ -119,9 +120,10 @@ const chatCompleteRoute = createObservabilityAIAssistantServerRoute({
       signal: controller.signal,
       resources,
       client,
+      appContexts,
     });
 
-    const response$ = await client.complete({
+    const response$ = client.complete({
```

Comment:
Thanks! This dangling await has been triggering severe OCD!

Reply:
😄 @miltonhultgren was complaining about it as well